ui/src/components/EntityModal/EntityModal.test.tsx

```diff
@@ -87,26 +150,72 @@ describe('EntityModal - Oauth oauth', () => {
         renderModalWithProps(props);
         const oauthTextBox = getDisabledOauthField();
         expect(oauthTextBox).toBeInTheDocument();
-        expect(oauthTextBox?.getAttribute('disabled')).toBeNull();
+        expect(oauthTextBox?.getAttribute('disabled')).toBe('');
```

Comment:
```suggestion
        expect(oauthTextBox).toHaveAttribute('disabled');
```

the current version:
<img width="761" alt="image" src="https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/dce830aa-53e5-4b80-a455-70c3d738bca5">

my suggestion:
<img width="761" alt="image" src="https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/dac5661f-46da-44a2-82b7-3f0103ffc108">

Reply:
fixed

Reply:
but it is not...?

Reply:
yea sorry, somehow i missed 2 out of 8 lines, should be done now